docs/installation.rst

```diff
@@ -157,7 +171,7 @@ OpenQL sources for each release can be downloaded from github `releases <https:/
 
 ::
 
-    git clone https://github.com/QE-Lab/OpenQL.git
+    git clone https://github.com/QE-Lab/OpenQL.git --recursive
```

Comment:
why the new flag?

Reply:
If you just do `--recursive` during pull you don't have to do `git submodule update --init --recursive`. It's redundant, but so many of the errors I've heard from people so far are just due to them not doing the latter I figured some redundancy wouldn't hurt.